src/graphql/batches.js

```diff
@@ -67,6 +71,7 @@ query GET_BATCH ($id:ID!) {
     enrollment_type
     state
     medha_area
+    mode_of_payment
```

Comment:
the purpose of the variable `batchesFields` was so that it can be reused. I still see code is being repeated as we have to write `mode_of_payment` at two places. Can you keep all the batchFields at a single place and reuse wherever it's needed in this file?